brave/mumble/service.py

```diff
@@ -162,7 +163,7 @@ def authenticate(self, name, pw, certificates, certhash, certstrong, current=Non
                 # Define the registration date if one has not been set.
                 Ticket.objects(character__name=name, registered=None).update(set__registered=datetime.utcnow())
         
-            for tag in ('member', 'blue', 'guest', 'mumble'):
+            for tag in ('admin', 'member', 'blue', 'guest', 'mumble'):
```

Comment:
Yes.

I didnt touch it as I marked usage of core groups as deprecated in a future commit https://github.com/kiu/mumble/commit/3f36c0a35362fd306e21c09d8f7ab0c04c9a649d#diff-c4e0646447c00f46082bb76ae59ffa61R35

But I can extract that.